sysutils/portfile-gen/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                portfile-gen
+version             0.2
+revision            0
+categories          sysutils macports
+maintainers         {gwmail.gwu.edu:egall @cooljeanius} openmaintainer
+platforms           darwin
+license             BSD
+supported_archs     noarch
+
+description         Generate a basic template Portfile given a few bits \
+                    of information
+long_description    ${description}.
+homepage            https://github.com/macports/macports-contrib/blob/master/${name}
```

Comment:
This looks like a candidate for the `github` portgroup... ?

Reply:
OK so I'm trying to convert it to use that, but unfortunately the most recent release in the macports-contrib repo has a space in its name, which is giving me trouble: https://github.com/macports/macports-contrib/releases/tag/pypi2port-20230621

Reply:
wait actually nvm, I figured it out (pasting that url here helped!)

Reply:
ok, shifted to use the `github` portgroup in [d88d81e](https://github.com/macports/macports-ports/pull/20951/commits/d88d81e5de6ff7dafcc2b60d466ed9cb9d06299c).